pkg/didcomm/protocol/didexchange/persistence.go

```diff
@@ -123,3 +201,26 @@ func computeHash(bytes []byte) (string, error) {
 	hash := h.Sum(bytes)
 	return fmt.Sprintf("%x", hash), nil
 }
+
+// connectionKey computes key for connection record object
```

Comment:
the comment is not valid pls replace `connectionKey` with `connectionKeyPrefix`

Reply:
done